index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="main-div">
+        <!--HEADER SECTION-->
+          <section class="header-section">
+              <h4>Portfolio</h4>
+              <header class="toolbar">
```

Comment:
- [ ] You could make this header tag anything else for example a div tag, because we can only have one header tag in an HTML page.